### DISCUSSION TITLE
Trying to set consumer.offsetCommitIntervalMs: 10000, but it is not working. I can still see that mirrormaker2 is taking 60 seconds to commit the offset number at the destination. How to resolve the issue.

### DISCUSSION THREAD

**Discussion by yogeshgoel001** [strimzi/strimzi-kafka-operator#10216] [Q&A]
Trying to set consumer.offsetCommitIntervalMs: 10000, but it is not working. I can still see that mirrormaker2 is taking 60 seconds to commit the offset number at the destination. How to resolve the issue.

**Comment by scholzj** [comment]
Maybe instead of asking the same thing again and again, you can add more information:
* What configurations did you try
* The logs from when you tried it
* What is the `offsetCommitIntervalMs` configuration option you are talking about

**Reply by yogeshgoel001** [reply]
Mirrormaker2 commits the offset details after every 60 seconds. We need to reduce that time. So I have tried multiple Options for that. 

1.) 
config:
      auto.commit.enabled: false
      auto.commit.interval.ms: 10000
      auto.offset.reset: latest

2.) 
config:
      auto.commit.enabled: false
      consumer.offsetCommitIntervalMs: 10000
      auto.offset.reset: latest

But no success, can we reduce the offset commit time to make it faster.

**Comment by yogeshgoel001** [comment]
Kindly help with this case
